db/schema.rb

```diff
@@ -26,9 +39,6 @@
     t.string "current_sign_in_ip"
     t.string "last_sign_in_ip"
     t.string "confirmation_token"
-    t.datetime "confirmed_at"
```

Comment:
I do not see a migration that removes it from db

Reply:
It was removed by mistake. Added it back